system/lang/pl.json

```diff
@@ -39,7 +61,31 @@
     "Extreme": "Ekstremalny",
     "Epic": "Epicki",
     "Stat": "Statystyka",
+    "OracleTable": {
+      "ColumnLabel": {
+        "Roll": "Rzut",
+        "RollRange": "Zakres rzutu",
+        "DiceRollResult": "{dice} wynik rzutu",
+        "Result": "Wynik"
+      }
+    },
     "roll +x": "rzuć +{stat}",
+    "Roll +x": "rzuć +{stat}",
+    "roll method": {
+      "All": "rzuć all of {statList} (????)",
```

Comment:
:thinking:

Reply:
You asking about ???? I probably forgot to delete them :D. I will change it soon ;) Thanks for feedback :D